modules/olm-policy-fine-grained-permissions.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * applications/operators/olm-creating-policy.adoc
+
+[id="olm-policy-fine-grained-permissions_{context}"]
+= Fine-grained permissions
+
+OLM uses the service account specified in OperatorGroup to create or update the
+following resources related to the Operator being installed.
+
+* ClusterServiceVersion
+* Subscription
+* Secret
+* ServiceAccount
+* Service
+* ClusterRole and ClusterRoleBinding
+* Role and RoleBinding
+
+In order to confine Operators to a designated namespace, cluster administrators
+can start by granting the following permissions to the service account:
+
```

Comment:
It's important to mention that the Role below is just a generic example and more rules may be added to adequate a specific operator needs.